sim/src/node.rs

```diff
@@ -76,12 +76,15 @@ impl SimElement for Node {
                 for n in &self.neighbors {
                     if n == &dest {
                         have_dest = true;
-                        ret.push((rpc, dest));
+                        ret.push((rpc, dest.clone()));
```

Comment:
What makes this necessary?

Reply:
It's not - I previously had subsequent prints and forgot to revert this once I got rid of the prints.

Reply:
There is one clone left?